### PR TITLE
Assembler: Navigate to the Celebration step

### PR DESCRIPTION
**Pull request by arthur791004**
Related to #

## Proposed Changes

* Add the celebration step to the end of the Assembler flow

### Demo

**Theme Showcase**

https://github.com/Automattic/wp-calypso/assets/13596067/6e0fa37a-31e8-4f6d-a91d-54528a395e2b

**Setup Site Flow**

https://github.com/Automattic/wp-calypso/assets/13596067/99557a31-fca3-4253-8b2c-1b824cb70199

## Testing Instructions

**Theme Showcase**

* Go to the Theme Showcase
* Scroll down to select the Assembler CTA
* On the Assembler screen
  * Pick patterns
  * Pick styles
  * Activate
  * After the activation, you will see the Congrats step

**Setup Site Flow**

* Go to `/setup?siteSlug=<your_site>`
* Continue to the Design Picker
* Select `Design your own` next to the categories
* * On the Assembler screen
  * Pick patterns
  * Pick styles
  * Start adding content <-- Do we need to update the copy?
  * After your site is ready, you will see the Congrats step

## Pre-merge Checklist

- [x] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [x] https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md for your changes?
- [x] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [x] Have you checked for TypeScript, React or other console errors?
- [x] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [x] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [x] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-ajp-p2)?

### CODE REVIEW

File: test/e2e/specs/onboarding/ftme__site-assembler.ts
Comment (worldomonation) [inline test/e2e/specs/onboarding/ftme__site-assembler.ts]: **suggestion:blocking** - I'm going to strongly suggest we add or move the 30s timeout to the `waitForURL( /site-editor/ )` portion, because the loading can take much longer than the 10s default timeout.

Comment (arthur791004) [reply test/e2e/specs/onboarding/ftme__site-assembler.ts]: Okay 👍

**Review by miksansegundo — COMMENTED** [review]
👍 Works well.

Are we merging this right away or waiting for stakeholders' feedback after a walkthrough?  

There is a period missing at the end of `Now it’s time to edit your content.` and a missing margin.

Screenshot 2566-09-13 at 09 48 46

**Review by taipeicoder — APPROVED** [review]
Tested and works as intended 👍

**Comment by taipeicoder** [comment]
Let's have @lucasmendes-design review this PR before merging. Should early next week, before the walkthrough.

**Comment by arthur791004** [comment]
> There is a period missing at the end of Now it’s time to edit your content. and a missing margin.

Hmmm...the missing margin seems only on dev env 🥲

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/8933875).

**Review by fushar — COMMENTED** [review]
I've tested it and everything works perfectly.

> Start adding content <-- Do we need to update the copy?

Yes, I believe we should and I think it is a blocker to this PR. Otherwise the flow does not make sense. Changing to `Continue` should work, I think. cc: @lucasmendes-design

**Comment by autumnfjeld** [comment]
@lucasmendes-design  and @ianstewart   Lego thinks this issue is longer valid because we will be going forward with Launchpad at the end of the Assembler. Assembler with Launchpad i1 – Dotcom Design p9Jlb4-9ns-p2 Thus we should not add this celebration step.

~Although I am a little confused why this issue is called a "celebration" step.  It doesn't look like celebration, it looks like an "informative" step. @arthur791004 .     All the other "celebration" steps have celebratory messaging and sometimes confetti.~ EDIT: Nevermind.

image

**Comment by autumnfjeld** [comment]
EDIT: NEVERMIND. Putting this in the icebox.     Ian and @taipeicoder wisely decided the celbration is a redundant screen.

@Automattic/lego  ~Let's ship this.~

~It will be at least a month, likely a couple months before we ship Launchpad to users.~

**Comment by autumnfjeld** [comment]
This is the screen that this PRs introduces:

image

File: test/e2e/specs/onboarding/ftme__site-assembler.ts
Comment (worldomonation) [inline test/e2e/specs/onboarding/ftme__site-assembler.ts:127]: **suggestion:non-blocking** - I ran the spec locally where the redirection to `/celebration-step/` took more than 30s. 

I think it might be useful to have this timeout be set to a longer value.

Comment (worldomonation) [reply test/e2e/specs/onboarding/ftme__site-assembler.ts:127]: Two runs both failed:

```
  ● Site Assembler › Site Assembler › Land on the Celebration step

    page.waitForURL: Timeout 30000ms exceeded.
    =========================== logs ===========================
    waiting for navigation until "load"
    ============================================================

      125 | 		it( 'Land on the Celebration step', async function () {
      126 | 			await page.waitForURL( /processing/ );
    > 127 | 			await page.waitForURL( /celebration-step/, { timeout: 30 * 1000 } );
          | 			           ^
      128 | 		} );
      129 |
      130 | 		it( 'Click "Edit your content" and land on the Site Editor', async function () {

      at Object.waitForURL (specs/onboarding/ftme__site-assembler.ts:127:15)

Test Suites: 1 failed, 1 total
Tests:       1 failed, 1 skipped, 11 passed, 13 total
Snapshots:   0 total
Time:        59.184 s, estimated 120 s
```

**Comment by lucasmendes-design** [comment]
If I understood correctly, there is no action from my side to be taken here. Right? 😆

**Comment by arthur791004** [comment]
Closing it as it's not planned. See https://github.com/Automattic/wp-calypso/issues/80672 for more details.

**Pull request closed without merging**
